libherokubuildpack/src/buildpack_output/mod.rs

```diff
@@ -11,7 +11,7 @@
 //!
 //! output = output
 //!     .section("Ruby version")
-//!     .end_section();
+//!     .finish();
```

Comment:
[comment] In the voice of subzero 🥶 finish him 🥶. 

I'm fine with this. Originally I wanted the code to not have the "finish" repetition, but I can go either way. One name makes it easier to remember

Reply:
We could also suffix the `finish` or `end` functions with _what_ they finish. My thought process was that it's inconsistent to have "finish" and "end". From there, I thought that the suffixes aren't necessary. But not a hill I wanna die on. TLDR: I could live with `finish_section` or `end_section` as long as we use the same word across the whole API to denote the end of a state or the whole output.